packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -52,65 +52,83 @@ Therefore, when dealing with Custom Roles, these three entities always exist:
 
-  ``` http
-  https://${yourOktaDomain}/api/v1/users
-  ```
+`orn:${partition}:${service}:${tenantId}:${objectType}:${objectId}`
 
-* [All Groups](/docs/reference/api/groups/#list-groups)
+###### partition
 
-  ``` http
-  https://${yourOktaDomain}/api/v1/groups
-  ```
+The partition is specific to your Okta environment. The following are the supported partitions.
 
-* [All Apps](/docs/reference/api/apps/#list-applications)
+| Partition               |  ORN partition value  |
+| ----------------------- | --------------------- |
+| Preview environments    | oktapreview           |
+| Production environments | okta                  | 
 
-  ``` http
-  https://${yourOktaDomain}/api/v1/apps
-  ```
+###### service
 
-* [A specific Group](/docs/reference/api/groups/#get-group)
+  The service that the resource belongs to. Each resource belongs to only one service.
+  The following are the supported services.
 
-  ``` http
-  https://${yourOktaDomain}/api/v1/groups/${targetGroupId}
-  ```
+| Service           |  ORN service value    |
+| ----------------- | --------------------- |
+| Directory         | directory             | 
+| Identity Provider | idp                   |
 
-* [All Users within a specific Group](/docs/reference/api/groups/#list-group-members)
+###### tenantId
 
-  ``` http
-  https://${yourOktaDomain}/api/v1/groups/${targetGroupId}/users
-  ```
+The identifier for the tenant that is using the service. This is typically your [org ID](/docs/reference/api/org/#org-setting-properties).
 
-* [All Apps of specific type](/docs/reference/api/apps/#list-apps-by-name)
+###### objectType
 
-  ``` http
-  https://${yourOktaDomain}/api/v1/apps/?filter=name+eq+\"${targetAppType}\"
-  ```
+The object type that is specific to the service. For example, object types `groups` or `users` are used for the `directory` service. To see examples of object types see
+[Supported Resources](#supported-rest-url-resources).
 
-* [A specific App](/docs/reference/api/apps/#get-application)
+###### objectId
 
-  ``` http
-  https://${yourOktaDomain}/api/v1/apps/${targetAppId}
-  ```
+The object's identifier. To see examples of objectIds see [Supported Resources](#supported-rest-url-resources).
+
+###### contained_resources
+
+`contained_resources` is an ORN property that indicates to target all resources within the container resource. For example,
+
+`orn:okta:directory:${yourOrgId}:groups:123:contained_resources`
+
+Group 123 is the example container resource. Since `:contained_resources` is specified, the resource includes the users in the group, rather than the group itself.
+
+Not all resources support this property, see [Supported Resources](#supported-rest-url-resources) for container resources.
+#### REST URL
+
+If the resource has a corresponding Okta API, you can specify the resource by their REST URL. Use the [ORN format](#orn-format) to specify resources that don't have corresponding Okta APIs.
+
+#### Supported REST URL resources
```

Comment:
```suggestion
#### Supported resources
```
Sorry, I think there used to be a table with just REST URLs here. There's both ORN and REST resources in the table below, so is it ok if I remove "REST URL" from the title?